add quickstart with flightdata and Tableau (#5)
hologres/hologres-doc: doc/source/quickstart/quickstart-Tableau.md

# Quickstart with Tableau

## Tableau：

> Tableau is an extremely powerful, secure and flexible end-to-end data analysis platform, providing a complete set of functions from connection to collaboration. Tableau Desktop is a visual data management tool of Tableau, which can provide real-time visual analysis by dragging and dropping, and quickly obtain practical results.

Reviewer: Tableau is a visual data management tool, which can provide visual analysis by dragging and dropping.

## Tableau Visualization While Connect With Hologres

> While we are doing query with Hologres, a lot of BI tools could be used to show us visualized query result  only need a few steps which can make the query result more understandable, below we will example some queries to show u how easy of visualization by using Tableau.

Reviewer: ```suggestion
Below we will example some queries to show you how easy to visualize data in Hologres via Tableau.
```
